Userland/Services/WindowServer/WindowManager.cpp

```diff
@@ -83,6 +83,7 @@ void WindowManager::reload_config()
 
     m_double_click_speed = m_config->read_num_entry("Input", "DoubleClickSpeed", 250);
     m_buttons_switched = m_config->read_bool_entry("Mouse", "ButtonsSwitched", false);
+    m_natural_scroll = m_config->read_bool_entry("Mouse", "NaturalScrolling", false);
```

Comment:
Let's use `NaturalScroll` here to match the name (or `m_natural_scrolling` if you prefer)